### PR TITLE
[WIP] Update GIMP and GEGL

### PR DESCRIPTION
**Pull request by unspecd**

### CODE REVIEW

**Comment by sgn** [comment]
Please bump revision for `gimp` `gmic` `gnome-photos` and `ufraw`, the build system should tell you that.

**Comment by ericonr** [comment]
Or make this PR and #25505 into a single one.

**Comment by ev-ermakov** [comment]
@sgn, @ericonr, please check if everything is looking fine.

**Review by fosslinux — APPROVED** [review]

**Comment by ev-ermakov** [comment]
`ufraw` fails to build (with xbps-src):
```
  CCLD     ufraw-gimp
  CCLD     ufraw
/usr/bin/ld: cannot find -lgimpui-2.0
/usr/bin/ld: cannot find -lgimpwidgets-2.0
/usr/bin/ld: cannot find -lgimpmodule-2.0
/usr/bin/ld: cannot find -lgimp-2.0
/usr/bin/ld: cannot find -lgimpmath-2.0
/usr/bin/ld: cannot find -lgimpconfig-2.0
/usr/bin/ld: cannot find -lgimpcolor-2.0
/usr/bin/ld: cannot find -lgimpbase-2.0
collect2: error: ld returned 1 exit status
```
But I can build it from source by hand.

**Comment by ev-ermakov** [comment]
> `ufraw` fails to build (with xbps-src):

Ah, I just forgot to switch branch before starting a build.

**Comment by biopsin** [comment]
heads up when gegl version bump: 
just tested against gegl-0.4.28 and gimp-2.10.22 fails to start with a popup
```
GEGL operation missing!
GIMP requires the GEGL operation "gegl:introspect".
This operation cannot be found. Check your
GEGL install and ensure it has been compiled
with any dependencies required for GIMP.
```
gegl-0.4.26 works fine tho

diff configure.log
```
< Source dir: /builddir/gegl-0.4.26
< Build dir: /builddir/gegl-0.4.26/build
> Source dir: /builddir/gegl-0.4.28
> Build dir: /builddir/gegl-0.4.28/build
< Project version: 0.4.26
> Project version: 0.4.28
> Program dot found: NO
> Run-time dependency maxflow found: NO (tried pkgconfig and cmake)
< Program dist-script.sh found: YES (/builddir/gegl-0.4.26/tools/dist-script.sh)
> Program dist-script.sh found: YES (/builddir/gegl-0.4.28/tools/dist-script.sh)
>   dot:             false
>   maxflow:         false
```

**Comment by fosslinux** [comment]
ping

File: common/shlibs [hunk 1]
```diff
@@ -2271,9 +2271,9 @@ libgegl-0.3.so.0 gegl3-0.3.0_1
 libgegl-npd-0.3.so gegl3-0.3.0_1
 libgegl-sc-0.3.so gegl3-0.3.0_1
 libprocps.so.8 procps-ng-3.3.16_1
-libgegl-0.4.so.0 gegl-0.4.16_1
-libgegl-npd-0.4.so gegl-0.4.16_1
-libgegl-sc-0.4.so gegl-0.4.16_1
+libgegl-0.4.so.0 gegl-0.4.26_1
+libgegl-npd-0.4.so gegl-0.4.26_1
+libgegl-sc-0.4.so gegl-0.4.26_1
```
Comment (ericonr) [inline common/shlibs:2276]: This doesn't make sense. The soname should have changed...

**Comment by q66** [comment]
also let's hold this until the locale mess is fixed (gimp is one of the affected things)

**Comment by biopsin** [comment]
sync with the babl update too https://github.com/void-linux/void-packages/pull/26833

**Comment by ericonr** [comment]
Ok, I'm taking this over in #27511

**Pull request closed without merging**